tests/data_asset/test_data_asset_util.py

```diff
@@ -52,7 +52,7 @@ def test_recursively_convert_to_json_serializable(tmp_path):
         # 'np.complex64': np.complex64([10.9999999 + 4.9999999j, 11.2+7.3j]),
         # 'np.complex128': np.complex128([20.999999999978335216827+10.99999999j, 22.4+14.6j]),
         # 'np.complex256': np.complex256([40.99999999 + 20.99999999j, 44.8+29.2j]),
-        "np.str": np.unicode_(["hello"]),
+        "np.str": np.str_(["hello"]).replace("'", '"'),
```

Comment:
np.unicode_ or np.str_ introduce single quotes by default.
I had to replace the single quotes with double quotes to pass the tests

Reply:
I finally canceled this modification